demo/src/main/AndroidManifest.xml

```diff
@@ -20,6 +19,10 @@
       </intent-filter>
     </activity>
 
+    <meta-data
+      android:name="com.novoda.downloadmanager.MaxConcurrentDownloads"
+      android:value="1" />
```

Comment:
yeah, we were thinking about how can we demo both behaviours, I think the only way is providing 2 apps.